packages/account/src/Configs/get-status-badge-config.tsx

```diff
@@ -1,95 +1,72 @@
 import React from 'react';
 import { Text } from '@deriv/components';
-import { AUTH_STATUS_CODES, MT5_ACCOUNT_STATUS, TRADING_PLATFORM_STATUS, routes } from '@deriv/shared';
+import { ACCOUNT_BADGE_STATUS, TAccountBadgeStatus } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
-import { TAuthStatusCodes, TMT5AccountStatus } from '../Types/common.type';
-import { Link } from 'react-router-dom';
 
-const getStatusBadgeConfig = (
-    mt5_account_status: TMT5AccountStatus,
-    openFailedVerificationModal?: (selected_account_type: string) => void,
-    selected_account_type?: string,
-    setIsVerificationModalVisible?: (value: boolean) => void,
-    user_account_status?: { poi_status: TAuthStatusCodes; poa_status: TAuthStatusCodes }
-) => {
-    const BadgeTextComponent = <Text key={0} weight='bold' size='xxxs' color='warning' />;
-
-    switch (mt5_account_status) {
-        case MT5_ACCOUNT_STATUS.PENDING:
+const getStatusBadgeConfig = (account_status: TAccountBadgeStatus, onClickBanner?: () => void) => {
+    const handleOnClick = () => {
+        if (onClickBanner) {
+            onClickBanner();
+        }
+    };
+    switch (account_status) {
+        case ACCOUNT_BADGE_STATUS.PENDING:
             return {
                 text: (
                     <Localize
-                        i18n_default_text='<0>Pending verification</0>'
-                        components={[<Text key={0} weight='bold' size='xxxs' color='var(--status-warning)' />]}
+                        i18n_default_text='<0>In review</0>'
+                        components={[
+                            <Text
+                                key={0}
+                                weight='bold'
+                                size='xxs'
+                                color='var(--status-warning)'
```

Comment:
May be u can use color="warning"

Reply:
hi @likhith-deriv how does color="warning" be identified as its not defined in themes.scss 
we have defined --status-warning in themes.scss uniquely for the badges only

Reply:
warning doesnt have corresponding dark color. if we add it now, it may effect all places where we have used 'warning'